### ISSUE TITLE
i wanna know the difference between ACSII, Unicode, UTF-8

### ISSUE THREAD

**Issue by LeoTunero** [LeoTunero/first_side_project#11]
you told me not only use ACSII in my program.
I did the study but not really understand the difference between the ACSII, Unicode, UTF-8 (Unicode Transformation Format). 
would you explain this more specific?

**Comment by VunkLai** [comment]
### Oops, not only use ASCII? 
Using ASCII only. Do Not Use Chinese Characters in Your Codes.
In addition to data, for example, i18n, chinese name, etc.

### Do you read the reference in #8 ?
In short, ASCII has `control characters`  and `printable characters`.
The printable characters represent letters (English), digits, punctuation marks and a few miscellaneous symbols.

### About Unicode or UTF-8. 
They are two encoding standards.
You could skip the technology about encoding temporarily, or google that by yourself.

### Why i do not explain more?
When you see a new keyword or technology, you should learn it by yourself first. 
Than you should show me what you **read and learn**.
I will give you details based on what you have learned.

In other words, that is not a good practice if you ask me to explain directly.
I hope you develop learning ability instead of learn a certain technology.

**Comment by LeoTunero** [comment]
I prefer to ignore this issue temporarily
I should focus on something else first.
